utils/cooperations/coopsAggregateOptions.js

```diff
@@ -1,20 +1,25 @@
+const mongoose = require('mongoose')
 const getRegex = require('../getRegex')
 
 const coopsAggregateOptions = (params = {}, query) => {
   const { id } = params
-  const { skip = 0, limit = 5, status = '', order = 'updatedAt', orderBy = 'asc', search } = query
-
+  const { skip = 0, limit = 5, status = '', sort = '{ order: "updatedAt", orderBy: 1 }', search } = query
   const match = {}
-  const sort = {}
-
-  if (order) sort[order] = orderBy
 
   if (status) match.status = getRegex(status)
-  if (id) match.$or = [{ initiator: id }, { receiver: id }]
+  if (id) match.$or = [{ initiator: mongoose.Types.ObjectId(id) }, { receiver: mongoose.Types.ObjectId(id) }]

```

Comment:
do we need this mongoose.Types.ObjectId? just 'id' didn't work?

Reply:
just id field works only on id field